src/stories/Autocomplete/Hook/index.tsx

```diff
@@ -119,32 +119,41 @@ export function HooksTemplate(args) {
             if (!section?.data?.length) {
               return null;
             }
+
             const { type, displayName } = section;
-            let sectionName = section.displayName;
+            const sectionListingType = isCustomSection(section)
```

Comment:
Is there a way we can get this from getSectionProps since this same logic is being duplicated 
https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/128/files#diff-7f1c987b478566cc9cf82a2afc2ee7a7a150ec6047a7801e0ac01c410f7ce453R218

Reply:
We could. The reason why its included here is because we've rendered a separate `div` with the relevant classes in [Line 154](https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/128/files#diff-0c9a1e20dad0ff16de33515c80eb5efdfda99bebefc29a58edf246f5d017c5dfR154) so we kind of have to duplicate the logic to get the required values here.

Took a look at our history and it might be something we've included prior to creating `getSectionProps` so we might be good to omit this entirely from the story. @esezen would love to get your thoughts here since you [touched it previously](https://github.com/Constructor-io/constructorio-ui-autocomplete/commit/3ad2d46789d886a966c15a80c774d6c57d74fd91#diff-0c9a1e20dad0ff16de33515c80eb5efdfda99bebefc29a58edf246f5d017c5dfR145).

Reply:
In terms of passing this variable via `getSectionProps` itself, I thought `getSectionProps` is meant to contain only the attributes we're going to render into the HTML? I would be against adding something in specifically for our Storybook.